src/Interactive/Host/Interactive/Core/InteractiveHost.LazyRemoteService.cs

```diff
@@ -103,7 +100,7 @@ private async Task<InitializedRemoteService> TryStartAndInitializeProcessAsync(C
 
             private Task<RemoteService> TryStartProcessAsync(string hostPath, CultureInfo culture, CancellationToken cancellationToken)
             {
-                return Task.Run(() => Host.TryStartProcess(hostPath, culture, cancellationToken));
+                return Task.Run(() => Host.TryStartProcessAsync(hostPath, culture, cancellationToken));
```

Comment:
>TryStartProcessAsync [](start = 43, length = 20)

Now that you made TryStartProcess async we don't need to wrap it into another Task.Run. You can remove this method and call `Host.TryStartProcessAsync` directly.